packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -156,6 +156,34 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
         />
       );
       break;
+    case roomNotFoundPageTrampoline():
+      pageElement = (
+        <>
+          {
+            /* @conditional-compile-remove(rooms) */ <NoticePage
+              iconName="InvalidRoom"
+              title={locale.strings.call.roomNotFoundTitle}
+              moreDetails={locale.strings.call.roomNotFoundDetails}
+              dataUiId={'room-not-found-page'}
+            />
+          }
+        </>
+      );
+      break;
+    case notInvitedToRoomPageTrampoline():
```

Comment:
Possible to remove whole case block in conditional compilation? This becomes 'endCall' when it is stable, wondering if this will block the condition for following sentences. Adding less code for stable will make it safer